_includes/pbc/all/install-features/202204.0/marketplace/install-the-marketplace-merchant-portal-core-feature.md

```diff
@@ -0,0 +1,1255 @@
+| AclMerchantPortalMerchantUserRoleFilterPreConditionPlugin | Checks if the Symfony security authentication roles should be filtered out. |  | Spryker\Zed\AclMerchantPortal\Communication\Plugin\MerchantUser  |
+| MerchantUserUserRoleFilterPlugin | Filters `ROLE_BACK_OFFICE_USER` to prevent Merchant User login to Backoffice. |  | Spryker\Zed\MerchantUser\Communication\Plugin\SecurityGui  |
+| ProductViewerForOfferCreationAclInstallerPlugin | Provide `ProductViewerForOfferCreation` Roles with Rules and Groups to create on install. |  | Spryker\Zed\AclMerchantPortal\Communication\Plugin\MerchantUser  |
+
+**src/Pyz/Zed/Twig/TwigDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Twig;
+
+use Spryker\Zed\GuiTable\Communication\Plugin\Twig\GuiTableConfigurationTwigPlugin;
+use Spryker\Zed\ZedUi\Communication\Plugin\Twig\BooleanToStringTwigPlugin;
+use Spryker\Zed\ZedUi\Communication\Plugin\ZedUiNavigationTwigPlugin;
+use Spryker\Zed\Twig\TwigDependencyProvider as SprykerTwigDependencyProvider;
+
+class TwigDependencyProvider extends SprykerTwigDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Shared\TwigExtension\Dependency\Plugin\TwigPluginInterface>
+     */
+    protected function getTwigPlugins(): array
+    {
+        return [
+            new ZedUiNavigationTwigPlugin(),
+            new BooleanToStringTwigPlugin(),
+            new GuiTableConfigurationTwigPlugin()
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Application/ApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Application;
+
+use Spryker\Zed\Application\ApplicationDependencyProvider as SprykerApplicationDependencyProvider;
+use Spryker\Zed\GuiTable\Communication\Plugin\Application\GuiTableApplicationPlugin;
+
+class ApplicationDependencyProvider extends SprykerApplicationDependencyProvider
+{
+
+    /**
+     * @return array<\Spryker\Shared\ApplicationExtension\Dependency\Plugin\ApplicationPluginInterface>
+     */
+    protected function getApplicationPlugins(): array
+    {
+        return [  
+            new GuiTableApplicationPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Security/SecurityDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Security;
+
+use Spryker\Zed\Security\SecurityDependencyProvider as SprykerSecurityDependencyProvider;
+use Spryker\Zed\SecurityGui\Communication\Plugin\Security\UserSecurityPlugin;
+use Spryker\Zed\SecurityMerchantPortalGui\Communication\Plugin\Security\MerchantUserSecurityPlugin;
+use Spryker\Zed\User\Communication\Plugin\Security\UserSessionHandlerSecurityPlugin;
+
+class SecurityDependencyProvider extends SprykerSecurityDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Shared\SecurityExtension\Dependency\Plugin\SecurityPluginInterface>
+     */
+    protected function getSecurityPlugins(): array
+    {
+        return [
+            new UserSessionHandlerSecurityPlugin(),
```

Comment:
Plugin `UserSessionHandlerSecurityPlugin` is not mentioned above.

Reply:
It looks like this plugin should not be mentioned here and should already be installed within the `Spryker Core BO` feature.

Reply:
If that's the case we it's mentioned here at all?